app/models/spotlight/resources/oaipmh_mods_parser.rb

```diff
@@ -189,5 +193,55 @@ def transform_to_iiif_thumbnail(ids_uri)
       uri = uri.gsub(/\/full\/\d*,\d*\/0\/default.jpg/, '')
       uri += "/full/#{thumbnail_size},/0/native.jpg"
     end
+
+    # Spotlight v3.3.0
+    # Spotlight expects "exhibit-specific fields" (a.k.a. Exhibit#custom_fields) to not have
+    # a Solr suffix (e.g. _tesim, _ssim, etc.). This method assumes all non-configured fields
+    # are custom and thus removes their Solr suffix when adding them to the @item_sidecar hash.
+    # Configured fields are added as-is (Solr suffix included).
+    def assign_item_sidecar_data(field_name, value)
+      if configured_field_names.include?(field_name)
+        item_sidecar[field_name] = value
+      else
+        custom_field_slug = field_name.sub(/_.*$/, '')
```

Comment:
With the original regular expression, any field that had multiple `_` would chop off all characters from the second `_` to the end of the string.

```suggestion
        custom_field_slug = field_name.sub(/_[^_]$/, '')
```

Below is a quick script to demonstrate:

```ruby
ORIGINAL_REGEXP = /_.*$/
MODIFIED_REGEXP = /_[^_]+$/
[ORIGINAL_REGEXP, MODIFIED_REGEXP].each do |regexp|
  puts "\nWith #{regexp.inspect}"
  [
    ["title_ssim", "title"],
    ["my_title_ssim", "my_title"]
  ].each do |given, expected|
    puts "Given: #{given}"
    puts "\tExpected: #{expected}"
    puts "\tActual: #{given.sub(regexp, '')}"
  end
end
```